src/custom/theme/cowSwapTheme.tsx

```diff
@@ -93,7 +93,11 @@ function themeVariables(colorsTheme: Colors) {
       boxShadow: `6px 6px 0px ${colorsTheme.black}`,
       borderRadius: '8px',
       border: `4px solid ${colorsTheme.black}`,
-      padding: '12px 6px 24px'
+      padding: '12px 6px 24px',
+      maxWidth: {
```

Comment:
Unnecessary duplication of the `maxWidth` object here. But Typescript complains the it was missing (given its mandatory as per Types).

But my assumption was given we merge the objects from the baseTheme, those would be included? Needs to be reviewed to not have to require adding `maxWidth` for the cowSwapTheme.tsx.